core/modules/dashboard/includes/block.taxonomy.inc

```diff
@@ -98,6 +104,7 @@ class DashboardTaxonomyBlock extends Block {
               backdrop_render($operations),
             ),
           );
+          unset($links);
```

Comment:
Instead of `unset()`ing `$links` here, can we add a `$links = array();` after `$terms = ...`?

Reply:
Done!! @klonos